homeassistant/components/bayesian/strings.json

```diff
@@ -14,5 +14,119 @@
       "name": "[%key:common::action::reload%]",
       "description": "Reloads bayesian sensors from the YAML-configuration."
     }
+  },
+  "config": {
```

Comment:
Is there a way to just do `"observation_selector": [%key:component::bayesian::config::step::observation_selector]` or does it only work for strings and not objects?